miso-web/src/main/webapp/scripts/list_experiment_run_partition.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK
+ * MISO project contacts: Robert Davey @ TGAC
+ * *********************************************************************
+ *
+ * This file is part of MISO.
+ *
+ * MISO is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MISO is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MISO.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * *********************************************************************
+ */
+
+ListTarget.experiment_run_partition = {
+  name: "Run + Partition",
+  createUrl: function(config, projectId) {
+    throw "Must be created statically.";
+  },
+  createBulkActions: function(config, projectId) {
+    return [];
+  },
+  createStaticActions: function(config, projectId) {
+    return [];
+  },
+  createColumns: function(config, projectId) {
+    return [
+        ListUtils.idHyperlinkColumn("Name", "run", "run.id", function(pair) {
```

Comment:
Column title is "Name," but it shows the Run ID. Rename "Run Name" for clarity and show the name or rename "Run ID"